PowerEditor/installer/nsisInclude/gobalDef.nsh

```diff
@@ -34,6 +34,12 @@
 !define VERSION_MAJOR 7
 !define VERSION_MINOR 0
 
+!define CompanyName		"Don HO don.h@free.fr"
+!define Description		"Notepad++ : a free (GNU) source code editor"
+!define Version 		"7.0.0.0"
```

Comment:
I think he ment using something like ...
``` NSIS
!define Version 		VERSION_MAJOR "." VERSION_MINOR
```
instead of ...
``` NSIS
!define Version 		"7.0.0.0"
```